tests/robot/suites/crud/loopback_crud.robot

```diff
@@ -0,0 +1,89 @@
+*** Settings ***
+Library      OperatingSystem
+#Library      RequestsLibrary
+#Library      SSHLibrary      timeout=60s
+#Library      String
+
+Resource     ../../variables/${VARIABLES}_variables.robot
+
+Resource     ../../libraries/all_libs.robot
+
+Suite Setup       Testsuite Setup
+Suite Teardown    Testsuite Teardown
+Test Setup        TestSetup
+Test Teardown     TestTeardown
+
+*** Variables ***
+${VARIABLES}=       common
+${ENV}=             common
+${NAME_LOOP1}=      vpp1_loop1
+${NAME_LOOP2}=      vpp1_loop2
+${MAC_LOOP1}=       12:21:21:11:11:11
+${MAC_LOOP1_2}=     13:21:21:11:11:11
```

Comment:
niektore mac su rezervovane a vpp s nimi moze mat problem. treba pouzit pre istotu len tieto:
There are actually 4 sets of Locally Administered Address Ranges that can be used on your network without fear of conflict, assuming no one else has assigned these on your network:

x2-xx-xx-xx-xx-xx
x6-xx-xx-xx-xx-xx
xA-xx-xx-xx-xx-xx
xE-xx-xx-xx-xx-xx
Replacing x with any hex value.
               
Useful translation of the wikipedia version: "Universally administered and locally administered addresses are distinguished by setting the second least significant bit of the most significant byte of the address. If the bit is 0, the address is universally administered. If it is 1, the address is locally administered. In the example address 02-00-00-00-00-01 the most significant byte is 02h. The binary is 00000010 and the second least significant bit is 1. Therefore, it is a locally administered address.[3] The bit is 0 in all OUIs." – Adam Brand Jul 14 '09 at 21:11
add a comment